doc/classes/CanvasItem.xml

```diff
@@ -634,7 +634,7 @@
 			The texture filtering mode to use on this [CanvasItem].
 		</member>
 		<member name="texture_repeat" type="int" setter="set_texture_repeat" getter="get_texture_repeat" enum="CanvasItem.TextureRepeat" default="0">
-			The texture repeating mode to use on this [CanvasItem].
+			The texture repeating mode to use on this [CanvasItem]. It affects what happens when the texture is sampled outside its size, e.g. by setting bigger [member Sprite2D.region] than the texture or setting [Polygon2D] UV points outside the texture.
```

Comment:
This is a good opportunity to add a meniton of what's causing https://github.com/godotengine/godot/issues/102965:

```suggestion
			The texture repeating mode to use on this [CanvasItem]. It affects what happens when the texture is sampled outside its size, e.g. by setting bigger [member Sprite2D.region] than the texture or setting [Polygon2D] UV points outside the texture.
			[b]Note:[/b] [TextureRect] does not follow [member texture_repeat], as it uses its own texture repeating implementation.
```